### DISCUSSION TITLE
Take Control takes me to the meshcentral login screen - After 0.12.3 Update

### DISCUSSION THREAD

**Discussion by johnnyq** [amidaware/tacticalrmm#1061] [General]
After updating to the latest 0.12.3 When I hit take control it take me to the meshcentral login screen.
I am able to login with my mesh username and password just fine but it appears the login token key no longer works.
I ran node /meshcentral/node_modules/meshcentral --logintokenkey

to verify I had the right token under Setting Global setings MeshCentral and it matched

Tried Deleting the settings fields out of global settings and then re adding them with a no go

is this a known bug

It affected both my instances my test machine and production

**Comment by dinger1986** [comment]
yes know issue, it will be resolved ASAP, devs are on the case now.

**Reply by johnnyq** [reply]
Fantastic Thank you! @dinger1986

**Reply by dinger1986** [reply]
and there you have it :) https://github.com/amidaware/meshctrl/releases/tag/v0.1.14, itll make its way through shortly to TRMM

**Reply by johnnyq** [reply]
Wow Thats just too fast!! 

Should I just run ./update.sh --force?

**Reply by dinger1986** [reply]
thats just the meshctrl part, not TRMM itself yet

**Comment by johnnyq** [comment]
oh okay, so ill hold off until the latest TRMM. Thank you!

**Comment by wh1te909** [comment]
fixed in 0.12.4

**Reply by johnnyq** [reply]
Super!!! Thank you @wh1te909
